src/blueteam/Archon.java

```diff
@@ -2,34 +2,40 @@
 
 import battlecode.common.Direction;
 import battlecode.common.GameActionException;
-import battlecode.common.MapLocation;
 import battlecode.common.RobotController;
+import battlecode.common.RobotInfo;
 
 public class Archon extends Robot {
 
+	private final int desiredNumberOfGardeners = 10;
```

Comment:
you are right. I think we should further move these constants to some commnon config, so we could tune our system more easily.